scripts/alert_emails/misc/delete-historical-docs.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Script to go through all the historical sent emails in Firestore and delete
+ * them to save space.
+ */
```

Comment:
I checked in these scripts for future reference, but we should never need to run them again and they're not particularly high quality. 🤷